contracts/access-controller/Cargo.toml

```diff
@@ -22,6 +22,7 @@ backtraces = ["cosmwasm-std/backtraces"]
 library = []
 
 [dependencies]
+cosmwasm-schema = { version = "1.1.5" }
```

Comment:
we have to move cosmwasm-schema from dev-dependency to dependency to use the new write_api! macro